pytorch_lightning/loggers/tensorboard.py

```diff
@@ -234,6 +234,9 @@ def save(self) -> None:
 
     @rank_zero_only
     def finalize(self, status: str) -> None:
+        self.close()
+
+    def close(self):
```

Comment:
If this is not part of the public logger API, you should prepend with `_`

Reply:
it should be removed completely and finalize should be used

Reply:
Move back !